### ISSUE TITLE
Pass model instance to event handlers

### ISSUE THREAD

**Issue by mdidkivskyi** [bem-node/promised-models#9]
It could be good to retrieve model via event handler, like this:
```
var myData = [1,2,4];
myModel.on('change', function (e, data) {
    console.log(e.type === 'change'); // true
    console.log(myModel === e.model); // true
    console.log(myData === data); // true
});
myModel.trigger('change', myData);
```

**Comment by mdidkivskyi** [comment]
After some discussion decided to use next:
```
myModel.on('change', function (model) {
    console.log(myModel === model); // true
});
```

**Comment by anton-kotenko** [comment]
Agree, good feature. Anyway model or at least value of attribute is required in callback.

**Comment by mdidkivskyi** [comment]
Implemented in #11
